analysis.go

```diff
@@ -34,10 +40,13 @@ type Event struct {
 	Type    string `json:"type"`
 }
 
+// Events is a representation of everything in the openshift event log for a
+// particular project.
 type Events struct {
 	Items []Event `json:"items"`
 }
 
+// DeploymentConfigs is a representation of the OpenShift DeploymentConfigs JSON.
```

Comment:
"DeploymentConfigs represents OpenShift deployment configs." ?

(it doesn't have to be JSON...)